packages/js/customer-effort-score/src/customer-feedback-modal/index.tsx

```diff
@@ -175,28 +176,32 @@ function CustomerFeedbackModal( {
 				/>
 			</div>
 
-			<Text
-				variant="subtitle.small"
-				as="p"
-				weight="600"
-				size="14"
-				lineHeight="20px"
-			>
-				{ secondQuestion }
-			</Text>
+			{ secondQuestion && (
```

Comment:
Instead of checking this condition twice, we could just combine the two with a fragment:
```js
			{ secondQuestion && (
				<>
					<Text
						variant="subtitle.small"
						as="p"
						weight="600"
						size="14"
						lineHeight="20px"
					>
						{ secondQuestion }
					</Text>

					<div className="woocommerce-customer-effort-score__selection">
						<RadioControl
							selected={ secondQuestionScore.toString( 10 ) }
							options={ options }
							onChange={ ( value ) =>
								onRadioControlChange(
									value as string,
									setSecondQuestionScore
								)
							}
						/>
					</div>
				</>
			) }
```

Reply:
That would create an extra empty <div> in the doom. If this is not a blocker I would prefer to keep it like this to maintain the original DOM estructure.